client/state/selectors/README.md

```diff
@@ -9,5 +9,49 @@ When adding a new selector to this directory, make note of the following details
 
 - Each new selector exists in its own file, named with [kebab case](https://en.wikipedia.org/wiki/Kebab_case) (dash-delimited lowercase)
 - There should be no more than a single default exported function per selector file
+- The JSDoc header for the default exported function should contain an alias tag with the name of the function in it and that should match the camelCase conversion of the filename itself
```

Comment:
> I think we should try to identify the default export by parsing the file.

Yeah this is just another one of those things where I wonder why we would build a custom parser if one mainly does it for us. I don't have any knowledge of how easy Babylon is to use so I can't really assess that route.

does Doctrine publish their specs? I couldn't find it. there are a couple things that make me hesitate about using it:
 - there's a lack of documentation
 - jsdoc3 _is_ the JSDoc spec while doctrine will have to track that to stay up-to-date. in other words, we're guaranteed to get the right parsing with jsdoc but I'm not sure we know how to evaluate the extent to which doctrine may or may not produce the right results.
 - _we_ have to find the right JSDoc blocks and match them up to their respective functions. this can work fine if we use something like Babylon, but it's another step we have to maintain